### PR TITLE
Sophos Central release note

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-06-27-collections.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - collection
+  - app catalog
+  - saas cloud
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We have revised the method of collecting endpoint data from the Sophos Central platform. Previously, endpoint data was combined with alerts and then sent to Sumo Logic. Now we have adopted a new approach where endpoint and alert data are sent separately to Sumo Logic. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source/).
```
Comment (JainM6) [inline blog-service/2023-06-27-collections.md:14]:
```suggestion
We've revised the method of collecting endpoint data from the Sophos Central platform. Previously, endpoint data were combined with alerts and sent to Sumo Logic. We have adopted a new approach where endpoint and alert data are sent separately to Sumo Logic. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source/).
```

Comment (JainM6) [reply blog-service/2023-06-27-collections.md:14]: Can we talk about what customers will achieve after implementing this new approach?  Just a benefit they will get form this. Add after "We have adopted a new approach where endpoint and alert data are sent separately to Sumo Logic.

File: blog-service/2023-06-27-collections.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Sophos Central (Collection)
```
Comment (kimsauce) [inline blog-service/2023-06-27-collections.md:2]:
```suggestion
title: Sophos Central Source (Collection)
```

File: blog-service/2023-06-27-collections.md (hunk 1 above)
Comment (JV0812) [reply blog-service/2023-06-27-collections.md]: Thanks Mega! I have updated the complete release note.

**Review by JainM6 — APPROVED** [review]
Looks good.

**Pull request merged**